app/src/test/java/dev/aurakai/auraframefx/ui/navigation/AppNavGraphTest.kt

```
@@ -0,0 +1,433 @@
+        composeTestRule.setContent {
+            navController = rememberNavController()
+            AppNavGraph(navController = navController)
+        }
+
+        // Test error handling in navigation
+        var errorOccurred = false
+        
+        composeTestRule.runOnIdle {
+            try {
+                navController.navigate("") // Empty route
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)